modules/apigee_edge_teams/src/Entity/Storage/TeamMemberRoleStorage.php

```diff
@@ -258,8 +258,10 @@ protected function doSave($id, EntityInterface $entity) {
   protected function doDelete($entities) {
     /** @var \Drupal\apigee_edge_teams\Entity\TeamMemberRoleInterface $entity */
     foreach ($entities as $entity) {
-      // See explanation in doSave().
-      Cache::invalidateTags($entity->getTeam()->getCacheTags());
+      if (($team = $entity->getTeam()) !== NULL) {
```

Comment:
Please document that this small sanity check allows uninstalling the module if the Apigee Edge connection would be broken.